signer/src/bitcoin/validation.rs

```diff
@@ -482,6 +527,11 @@ impl BitcoinTxValidationData {
         let tx = &self.tx;
         let tx_fee = self.tx_fee;
         let sbtc_limits = &self.sbtc_limits;
+        let estimated_fee = self.signer_estimated_fee;
+
+        if tx_fee.to_sat() > highest_acceptable_fee_sats(estimated_fee) {
+            return false;
+        }
```

Comment:
The only trouble with this is that it's not clear why the the transaction is invalid when looking at the data in the signers' database. I wonder if we should do something else here instead. Maybe we could change the result for the signers' UTXO to be something other than `Ok` when the fee is too high? Anyone have any opinions here?